libbpf-tools/ksnoop.bpf.c

```diff
@@ -313,6 +317,18 @@ static int ksnoop(struct pt_regs *ctx, bool entry)
 		return 0;
 	trace->pid = currpid;
 
+	/*  get the pid namespace by following task_active_pid_ns(),
+	 *  pid->numbers[pid->level].ns
+	 */
+	task = (struct task_struct *)bpf_get_current_task();
+	pid = (struct pid *)BPF_CORE_READ(task, thread_pid);
+	level = (unsigned int)BPF_CORE_READ(pid, level);
+	bpf_core_read(&upid, sizeof(upid), &pid->numbers[level]);
+	trace->pid_ns_id =
+	    (__u32)BPF_CORE_READ(upid.ns, ns.inum);
```

Comment:
Keep this in a single line.

Reply:
@chenhengqi 
Which tool/script is used for libbpf-tools code formating?